slam-libraries/viam-orb-slam3/orbslam_server_v1.cc

```diff
@@ -866,16 +878,18 @@ int FindFrameIndex(const std::vector<std::string> &filesRGB,
 
 // Make a filename to a specific location for a sensor with a timestamp
 // currently does not support millisecond resolution
-// TODO change time format to .Format(time.RFC3339Nano)
-// https://viam.atlassian.net/browse/DATA-277
 string MakeFilenameWithTimestamp(string path_to_dir, string camera_name) {
     std::time_t t = std::time(nullptr);
     char timestamp[100];
-    std::strftime(timestamp, sizeof(timestamp), "%FT%H_%M_%S", std::gmtime(&t));
+    std::strftime(timestamp, sizeof(timestamp), time_format.c_str(),
+                  std::gmtime(&t));
     // Save the current atlas map in *.osa style
-    string path_save_file_name =
-        path_to_dir + "/" + camera_name + "_data_" + timestamp + ".0000.osa";
-    return path_save_file_name;
+    auto find_underscores = time_format.find("_");
+    if (find_underscores != string::npos) {
+        return path_to_dir + "/" + camera_name + "_data_" + timestamp +
+               ".0000.osa";
+    }
+    return path_to_dir + "/" + camera_name + "_data_" + timestamp + "Z.osa";
```

Comment:
Is the "Z" not already included in the timestamp?

Reply:
good catch, forgot to fix this here